netjsonconfig/backends/zerotier/schema.py

```diff
@@ -188,6 +191,32 @@
                         "items": {"type": "object"},
                         "propertyOrder": 17,
                         "description": "Array of network rule objects",
+                        # This is the default rule set
+                        # that allows IPv4 and IPv6 traffic
+                        # It is based on the default
+                        # network configuration from ZeroTier Central
+                        "default": [
+                            {
+                                "etherType": 2048,
```

Comment:
What is this `etherType`?
Given how confusing it looks, I'd be in favor of adding documentation link in the code itself as well! 😅

Reply:
Thanks for pointing it out @atb00ker!
I will include the [link to the flow rules engine](https://docs.zerotier.com/zerotier/rules/) in the following section of the code and update the commit description as well.